### PR TITLE
empty on monitor and next empty flags

### PR DESCRIPTION
**Pull request by wagyourtail**
#### Describe your PR, what does it fix/add?

add a selector to select the first empty workspace on the monitor. and a flag to only select empty workspaces with a higher id.

flags work by suffix'ing empty
ie.
`emptynm` for next empty on the same monitor
`emptym` for first empty on same monitor

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)

so basically, I want this so when I hit my fullscreen bind the current window moves to the next empty workspace on the same monitor, without my program just spamming `r+1` until it finds one.

#### Is it ready for merging, or does it need work?

yes.

### CODE REVIEW

**Review by Agent00Ming — APPROVED** [review]
Works as described. Very convenient addition especially for larger multi-monitor setups. :+1:
